main.cpp

```diff
@@ -15,8 +15,59 @@ Create a branch named Part2
     You'll need to insert the Person struct from the video in the space below.
  */
 
+#include <string>
+#include <iostream>
 
+struct Person
+{
+    int age;
+    int height;
+    float hairLength;
+    float GPA;
+    unsigned int SATScore;
+    int distanceTravelled;
+
+    struct Foot
+    {
+        int length;
+        float width;
+        float shoeSize;
+        void stepForward();
+        int stepSize(int howLong);
+    };
+    Foot rightFoot;
+    Foot leftFoot;
+    void run(int howFast, bool startWithLeftFoot);
+    
+};
+void Person::Foot::stepForward()
+{
+    length = 1.0f;
+}
```

Comment:
every time this is called, you're assigning the value.  do something else; 
give `length` an initial value of 1 (it is an integer, not a float too) where you declare it in the class.
- [ ] revise